apps/passport-client/test/util.spec.ts

```diff
@@ -14,12 +14,16 @@ describe("util", async function () {
         location: {
           origin: mockOrigin
         }
-      }
+      },
+      configurable: true,
+      writable: true
     });
   });
 
   afterEach(() => {
-    delete global.window.location;
+    Object.defineProperty(global, "window", {
+      value: { location: undefined }
+    });
```

Comment:
just curious but which test required this change?

Reply:
Can't `delete` a non-optional property in strict mode (`global.window.location`).